features/find_car.feature

```diff
@@ -0,0 +1,286 @@
+      | transmission       | Manual              | Manual          |
+      | seller-type        | private             | Private adverts |
+      | insuranceGroup     | 20U                 | Up to 20        |
+
+  Scenario Outline: Checking 'Max engine size' filter
+    When user fill in Postcode field with correct data
+    And select <filter_value_option> from <filter_name> filter
+    And click the Search cars button
+    Then all cars with <filter_text> <filter_option> or less are shown
+
+    Examples:
+      | filter_name               | filter_value_option |  filter_text    | filter_option   |
+      | maximum-badge-engine-size | 1.4                 | Engine size     | 1.4L            |
+
+  Scenario Outline: Checking 'Mileage' filter
+    When user fill in Postcode field with correct data
+    And select <filter_value_option> from <filter_name> filter
+    And click the Search cars button
+    Then cars with <filter_text> no more than <filter_option> are shown
+
+    Examples:
+      | filter_name     | filter_value_option | filter_text | filter_option      |
+      | maximum-mileage | 15000               | Mileage     | Up to 15,000 miles |
+
+  Scenario Outline: Checking 'Min seats' filter
+    When user fill in Postcode field with correct data
+    And select <filter_value_option> from <filter_name> filter
+    And click the Search cars button
+    Then cars which have <filter_text> no less than <filter_option> are shown
+
+    Examples:
+      | filter_name   | filter_value_option | filter_text | filter_option   |
+      | minimum-seats | 5                   | Seats       | 5               |
+
+  Scenario Outline: Checking 'Max seats' filter
+    When user fill in Postcode field with correct data
+    And select <filter_value_option> from <filter_name> filter
+    And click the Search cars button
+    Then cars which have <filter_text> no more than <filter_option> are shown
+
+    Examples:
+      | filter_name   | filter_value_option | filter_text | filter_option   |
+      | maximum-seats | 5                   | Seats       | 5               |
+
+  Scenario Outline: Checking 'Min seats' and 'Max seats' filters together
+    When user fill in Postcode field with correct data
+    And select <filter_min_value_option> from <filter_min_name> filter
+    And select <filter_max_value_option> from <filter_max_name> filter
+    And click the Search cars button
+    Then cars with number of <filter_text> no less than <filter_min_option>
+    And number of <filter_text> no more than <filter_max_option> are shown
+
+    Examples:
+      | filter_min_name | filter_max_name | filter_min_value_option | filter_max_value_option | filter_text | filter_min_option | filter_max_option |
+      | minimum-seats   | maximum-seats   | 5                       | 7                       | Seats       | 5                 | 7                 |
+
+  Scenario Outline: Checking 'Reset search' link
```

Comment:
Incorrect test. Before resetting something, you should set a filter first. I don't see this action here. Also, I suggest to add steps that will select all filters, than click `Reset` and check that all filters has been reseted.